go/runtime/registry/config.go

```diff
@@ -143,6 +143,15 @@ func (m RuntimeMode) IsClientOnly() bool {
 	return false
 }
 
+// HasLocalStorage returns true iff the mode is one that has local storage.
+func (m RuntimeMode) HasLocalStorage() bool {
```

Comment:
This new method should also now be used in the storage worker constructor to make it consistent:
https://github.com/oasisprotocol/oasis-core/blob/d8e352bd7f1cca56b79bb21431da9bf97dbb22d2/go/worker/storage/worker.go#L45-L52

Reply:
Updated